src/electron/preload.ts

```diff
@@ -0,0 +1,27 @@
+// Copyright 2022 The Outline Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Directly import @sentry/electron main process code.
+// See: https://docs.sentry.io/platforms/javascript/guides/electron/#webpack-configuration
+
+import * as sentry from "@sentry/electron/dist/renderer";
```

Comment:
```suggestion
import * as sentry from "@sentry/electron/renderer";
```

Reply:
Actually, this should be in the electron_main.ts, not here

Reply:
Yup, I tried that, still not working.

Reply:
Not applicable any more.